tests/rbd/readonly_namespace_image_migration.py

```diff
@@ -70,7 +72,20 @@ def image_migrate_and_verify(
         image_name=image,
         namespace=namespace,
     )
-    out, err = rbd.migration_prepare(src_spec1, dest_spec1, client_id=client_id)
+
+    out, err = rbd.rbd_bench(
+        imagespec=src_spec1, client_id=client_id, all=True, check_ec=False
+    )
+    if "Operation not permitted" in err:
+        log.error(f"{err}")
```

Comment:
We should not be flushing output to log.error as actually its not error from test case perspective ?
cc @manasagowri

Reply:
removing log.error